controllers/details.js

```diff
@@ -1,13 +1,16 @@
 'use strict';
 
 var api = require('../api');
+var helpers = require('../lib/helpers');
 
 module.exports = function renderDetails(req, res) {
   if (req.params && req.params.sysnum) {
-    var username = req.headers['X-Auth-Username'] || req.headers['x-auth-username'];
+    const username = req.headers['X-Auth-Username'] || req.headers['x-auth-username'];
+    const canRedirectToResults = req.query ? req.query.multipleResults || false : false;
```

Comment:
Could probably avoid the ternary here if you wanted:
const canRedirectToResults = req.query && req.query.multipleResults || false;

Or instead guarantee a boolean:
const canRedirectToResults = req.query && req.query.multipleResults ? true : false;